content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty. It is possible that your browser cookies
+      settings policy did prevent it to load.
+      Note that this policy does also change if you are connected in "Incognito" mode on your
+      browser.
+
+      To fix this issue, you can configure your browser to always allow cookies on Odoo's plugin page.
+
+      For Chrome in particular you can do so by following the guide at:
+      `https://support.google.com/chrome/answer/95647 <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
+      then add `download.odoo.com` in the list of `Sites that can always use cookies`.
+
+      .. image:: outlook/cookie-fix.png
+         :align: center
+         :alt: Chrome cookie configuration to fix the empty panel issue
+
+      Once done, the panel needs to be open again.
```

Comment:
Fixed a small grammar mistake.

`Once done, the panel needs to be opened again.`

Also, which panel are you referring to? It would be good to specify.